libp2p/protocols/pubsub/floodsub.nim

```diff
@@ -57,10 +57,10 @@ proc addSeen*(f: FloodSub, saltedId: SaltedId): bool =
 proc firstSeen*(f: FloodSub, saltedId: SaltedId): Moment =
   f.seen.addedAt(saltedId)
 
-proc handleSubscribe*(f: FloodSub,
-                      peer: PubSubPeer,
-                      topic: string,
-                      subscribe: bool) =
+proc handleSubscribe(f: FloodSub,
+                     peer: PubSubPeer,
+                     topic: string,
+                     subscribe: bool) =
```

Comment:
it might be better to follow this pattern https://github.com/vacp2p/nim-libp2p/pull/1095/files#diff-94a6536a89bc744b23b27b6d4577eee57ed6e4ed0b1f13f077b9db22c16ae8b8R280-R284. @arnetheduck how are you formatting it? Let's add it to the readme so we all share the same formatting rules.

Reply:
my preference is to format code with https://github.com/arnetheduck/nph which broadly follows https://status-im.github.io/nim-style-guide/formatting.style.html